sections/method/Clustering-PageRank.tex

```diff
@@ -1,5 +1,8 @@
 \section{Cluster-based Random Walk}\label{sec:cluster_pagerank}
-To perform random walk on a document set, such as Nordjyske, probabilities of jumping between documents is needed to perform each step. \vejleder[inline]{say why this is usefull}.
+To perform random walk on a document set, such as Nordjyske, probabilities of jumping between documents is needed to perform each step. 
+When performing a random walk, we get the articles that most important based on the information that the probabilities are based on. 
+If it was based on word frequency, a random walk would return an ordered list of articles, where the ordering is based on the frequency of a given word.
+ 
 We have chosen to use a similarity measure between each document as these probabilities.
 This is calculated by using the document-topic distributions from the \gls{lda}.
 These distributions can be arranged in a matrix $\theta$, which is described later.
```

Comment:
after this we can start on the ClusterRW subsection, although we should proabably have some text about personalization vector beforehand.